### PR TITLE
Add guide to switch Prometheus

### PR DESCRIPTION
**Pull request by benjaminhuo**
Signed-off-by: Benjamin <benjamin@yunify.com>

### CODE REVIEW

File: content/en/docs/faq/byop.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Bring your own Prometheus"
```
Comment (FeynmanZhou) [inline content/en/docs/faq/byop.md:2]: Replace the built-in Prometheus with your own, how about this title?

File: content/en/docs/faq/byop.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+keywords: "Monitoring, Prometheus, node-exporter, kube-state-metrics, KubeSphere, Kubernetes"
+description: "Use your own Prometheus stack for KubeSphere monitoring"
+
+Weight: 7100
+---
+
+KubeSphere comes with several pre-installed customized monitoring components including Prometheus Operator, Prometheus, Alertmanager, Grafana (Optional), various service monitors, node-exporter, kube-state-metrics. These components might already exist before you install KubeSphere, it's possible to use your own Prometheus stack setup in KubeSphere v3.0.0 .
```
Comment (Sherlock113) [inline content/en/docs/faq/byop.md:9]: Just one thing to confirm here. Regards these components mentioned in this paragraphs, will they be automatically installed if KubeSphere is installed with the minimum package. Do I need to enable components like logging or alerting to have them installed?

Comment (benjaminhuo) [reply content/en/docs/faq/byop.md:9]: they will be automatically installed with the minimum package

File: content/en/docs/faq/byop.md (hunk 1 above)
Comment (Sherlock113) [reply content/en/docs/faq/byop.md:2]: I agree

Comment (benjaminhuo) [reply content/en/docs/faq/byop.md:2]: BYO or bring your own is an idiom actually
image

**Comment by Sherlock113** [comment]
The process looks clear to me. No major wording issue. I will improve the overall wording after the guide is merged.

File: content/en/docs/faq/byop.md (hunk 1 above)
Comment (Sherlock113) [reply content/en/docs/faq/byop.md:2]: I see...I also noticed some expressions from Oracle and Azure. BYO is an idiom. Learned and thanks.

**Comment by benjaminhuo** [comment]
Can we get this online today?

**Review by rayzhou2017 — APPROVED** [review]
/lgtm

**Pull request merged**